src/Razor/test/Microsoft.AspNetCore.Razor.LanguageServer.Test/AutoInsert/RazorOnAutoInsertProviderTestBase.cs

```diff
@@ -19,11 +20,10 @@ public abstract class RazorOnAutoInsertProviderTestBase : LanguageServerTestBase
     {
         internal abstract RazorOnAutoInsertProvider CreateProvider();
 
-        protected void RunAutoInsertTest(string input, string expected, string character, int tabSize = 4, bool insertSpaces = true, string fileKind = default, IReadOnlyList<TagHelperDescriptor> tagHelpers = default)
+        protected void RunAutoInsertTest(string input, string expected, int tabSize = 4, bool insertSpaces = true, string fileKind = default, IReadOnlyList<TagHelperDescriptor> tagHelpers = default)
         {
             // Arrange
-            var location = input.IndexOf('|', StringComparison.Ordinal) + character.Length;
-            input = input.Replace("|", character, StringComparison.Ordinal);
```

Comment:
@NTaylorMullen this line was inserting the character at the same location where I am now inserting it as part of the test input. On the previous like you can see the location was also being adjusted by the length of `character`. I removed the adjustment and inlined the character insertion.